asminc/vic20.inc

```diff
@@ -66,16 +66,43 @@ VIC_COLOR       := $900F        ; Border and background color
 ; ---------------------------------------------------------------------------
 ; I/O: 6522 VIA1
 
-VIA1            := $9110
-VIA1_JOY        := $9111
-VIA1_DDRB       := $9112
-VIA1_DDRA       := $9113
+VIA1            := $9110        ; *** Deprecated ***
+VIA1_JOY        := $9111        ; *** Deprecated ***
+VIA1_PB         := $9110        ; Port register B
+VIA1_PA1        := $9111        ; Port register A
```

Comment:
I would like to see the "R" put into those names (`VIA1_PRB`, `VIA1_PRA1`, and `VIA1_PRA2`).  It would be consistent with other register names (such as `VIA1_IFR`) and the port names in other ".inc" files (such as `CIA1_PRB` in "c64.inc").

Reply:
ugh no, that would be totally unlike what they are commonly called

Reply:
yeah, terrible. the other headers should be fixed accordingly

Reply:
Guys, I'm lost here. Do we want VIA1_PB or VIA1_PRB in vic20.inc?

Reply:
> Guys, I'm lost here. [...]

So at least I'm not the only one ;-)

Reply:
my point is: naming things differently than the official datasheets do is a bad idea. and tacking an extra R to every define because its a register (doh, what else?) is questionable at best. oh well :)